File: Python/Search Insert Position.py
class Solution(object):
    def searchInsert(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: int
        """
        if nums is []:
            return 0
        else:
            for idx, num in enumerate(nums):
                if target <= num:
                    return idx
            else:
                return len(nums)

File: cpp/Search Insert Position.cpp
class Solution {
public:
    int searchInsert(vector<int>& nums, int target) {
        if (nums.size() == 0) {
            return 0;
        } else {
            for (size_t idx = 0; idx < nums.size(); idx++) {
                if (target <= nums[idx]) {
                    return idx;
                }
            }
            return nums.size();
        }
    }
};
